frontend-react/src/pages/Login.tsx

```diff
@@ -12,6 +12,15 @@ export function Login() {
     const { oktaAuth, authState, config } = useSessionContext();
     const location: Location<{ originalUrl?: string } | undefined> =
         useLocation();
+    const [searchParams] = useSearchParams();
+    const finalConfig = useMemo(
```

Comment:
Will these values get cleared on logout and on session expiration?

Reply:
They are only URL params collected on the login page. There is no persistence and natural login process will redirect the user after successful auth which won't have any params.